src/test/resources/trello/config.properties

```diff
@@ -0,0 +1,8 @@
+token=
+key=
+base_uri=https://api.trello.com/1
+boardName=First_board
```

Comment:
test.property is not a place for test data

Reply:
Where should I place this data ?

Reply:
in data layer

Reply:
Do u smth like this? 
<img width="923" alt="Screenshot 2021-09-20 at 13 48 16" src="https://user-images.githubusercontent.com/84539715/133990586-398abdb2-5709-4131-8fae-84593d89036b.png">

Reply:
It is a bit complicated for HW, but why not - try it 😃